1208-projects/user_edits_project.feature

```diff
@@ -0,0 +1,93 @@
+Feature: User edits Project
+
+    @xxx
+    Scenario: User sees the Project Wizard
+        Given User with access rights for Project
+        When they open the Project Wizard
```

Comment:
her er det fortsatt litt kryptisk med "User with access rights", som heller ikker er noe scenario.
dette kan kanskje funke?
```suggestion
        Given Privileged user sees Edit button for Project
        When they click the Edit Button
```

evt kan vi fjerne scenario for `Given User clicks the Create New Project Button` og bruke dette her:
```
Given User opens My Projects
When they click Create New Project Button
```